prospero-cli/src/main/java/org/wildfly/prospero/cli/CliMain.java

```diff
@@ -78,9 +78,10 @@ public static CommandLine createCommandLine(Console console, ActionFactory actio
         repoCmd.addSubcommand(new RepositoryRemoveCommand(console, actionFactory));
         repoCmd.addSubcommand(new RepositoryListCommand(console, actionFactory));
 
-        CommandLine installCmd = commandLine.getSubcommands().get(CliConstants.INSTALL);
-        installCmd.setHelpFactory(new CustomHelp.CustomHelpFactory());
-        commandLine.setHelpFactory(new CustomHelp.CustomHelpFactory());
+        // CommandLine installCmd = commandLine.getSubcommands().get(CliConstants.INSTALL);
```

Comment:
this can be removed?

Reply:
Sure. BTW the `CustomHelp` now remains unused, but maybe we fill find something to tweak yet.